db/migrate/20241217145653_add_publisher_ats_api_client_reference_to_vacancy_table.rb

```diff
@@ -0,0 +1,16 @@
+class AddPublisherAtsApiClientReferenceToVacancyTable < ActiveRecord::Migration[7.2]
+  disable_ddl_transaction!
```

Comment:
I'm not sure if this is the right strategy? The index create will be trivial (as there are no references), so I suspect it would be safer to keep the DDL transactions switched on and tell our library that we know what we're doing in this case

Reply:
maybe it needs a comment?